resources/php_ini.rb

```diff
@@ -0,0 +1,23 @@
+resource_name :php_ini
+
+default_action :add
+
+property :mode, String, default: '0644'
+property :options, Hash, default: {}, required: true
+
+action :add do
+  directory '/etc/php.d'
```

Comment:
I just noticed that this is going to create duplicate resources. A better way to do this is like this:
``` ruby
directory "/etc/php.d #{new_resource.name}" do
  path '/etc/php.d'
end
```